improver_tests/acceptance/SHA256SUMS

```diff
@@ -674,17 +674,17 @@ a699ad6697b5ffff8d0b7ef46c54ae46d545ba7141e30e351373b82cf998bb08  ./weighted_ble
 737ce86a68237cbeae61949de1f2edf8a0b40f33af7f630fbdc36e5534954703  ./weighted_blending/realizations/kgo.nc
 5468dd45409d3414a369bb6be7e21dd2f5c16f2a0c0d7f58438de51ef7f68acb  ./weighted_blending/spatial_weights/kgo/cycle.nc
 4b49f76a6eae71a549aedcacc5609b3c35e2ab3d052a6cb1b91bd6b53e8bf163  ./weighted_blending/spatial_weights/kgo/cycle_no_fuzzy.nc
-dbaf33c6e66874468fbb1dad4bcd2b0aeec9c4cbdeaec3163aff27fc0c4f537a  ./weighted_blending/spatial_weights/kgo/model.nc
-ce3fb86673de940a9acc8451bb3b7a9f5048d97638025f1b976856510a268024  ./weighted_blending/spatial_weights/kgo/model_no_fuzzy.nc
+9b6887ee2a8c1c5950e146e7f5ab97895c7627446a643cb2683a796093c29a09  ./weighted_blending/spatial_weights/kgo/model.nc
+8eaa4bdfcd9a01447c0c198482f4560e76020e85fb2266d52bfd46a8cea65371  ./weighted_blending/spatial_weights/kgo/model_no_fuzzy.nc
 45634420f7e4910b6e617c96ac50129973748f091617e2c6b65ed34f18fb71f5  ./weighted_blending/spatial_weights/nowcast_data/20181129T1000Z-PT0002H00M-lwe_precipitation_rate.nc
 b72b53e42fe91d4d8e4a9e121663f04284830f5827224b3937dc9cda7dd69652  ./weighted_blending/spatial_weights/nowcast_data/20181129T1000Z-PT0003H00M-lwe_precipitation_rate.nc
 8ba212c3657d008776d8731a0799dcc38f5507e59872e74650f24f460aa8afab  ./weighted_blending/spatial_weights/nowcast_data/20181129T1000Z-PT0004H00M-lwe_precipitation_rate.nc
 be1b5966899335bbd09c1932506d7dc43f541ef39e4bc15aa912438dc4dca1da  ./weighted_blending/spatial_weights/ukvx_data/20181129T1000Z-PT0002H00M-lwe_precipitation_rate.nc
 d08df3e0ecb68e7307fcf0085b263a7baecf4f811b3acee9f3a8f34616d13a03  ./weighted_blending/three_models/enukxhrly/20190101T0400Z-PT0004H00M-precip_rate.nc
-d197c8f5f1a08caddaa5efabac4a52d75185889d64d2c4b72cbba2f2c0f42a20  ./weighted_blending/three_models/kgo.nc
+a723b5cc5a2ba050a33dcc569ba67b93e8043d50bbd868cfcd065d000ae52b39  ./weighted_blending/three_models/kgo.nc
 30062d9c23527634a20cae6e7cf7a163aaee361b06ee800c8bea70a7a0f412ec  ./weighted_blending/three_models/nc/20190101T0400Z-PT0001H00M-precip_rate.nc
 850d48dfea1d1e5b696324f0e57a52fce3e3128008c58ccaca82e31598196c67  ./weighted_blending/three_models/ukvx/20190101T0400Z-PT0002H00M-precip_rate.nc
-63b4691e4d7e246b58e01c84f3c8cf82a6cdc7e14068ddd96418e8c0054e2bcf  ./weighted_blending/weights_from_dict/kgo.nc
+1eac7df64f49df69aabadc49c7ddaad7d2acd72dc775c99930a963bc6567cb03  ./weighted_blending/weights_from_dict/kgo.nc
 c2b71855f260544ca75aaa5900f3297b58cee1b3b64eec5805662e1927d5fdab  ./wet-bulb-temperature-integral/basic/input.nc
 651c80cd4ab7c720c59c6c0c2c4a2ade85aee81c2ab0c697b577ca87a36d57f2  ./wet-bulb-temperature-integral/basic/with_id_attr/kgo.nc
```

Comment:
Would there be any benefit to updating the inputs to the weather code tests? I'm inclined to say no.

Reply:
I've modified the basic test inputs to include a weight. The KGO is unchanged, demonstrating that the weights are thrown away as expected.